src/autoscaler/db/sqldb/policy_sqldb.go

```diff
@@ -145,3 +146,41 @@ func (pdb *PolicySQLDB) DeletePolicy(appId string) error {
 	}
 	return err
 }
+
+func (pdb *PolicySQLDB) GetCustomMetricsCreds(appId string) (string, string, error) {
+	var password string
+	var username string
+	query := "SELECT username,password from credentials WHERE id = $1"
+	err := pdb.sqldb.QueryRow(query, appId).Scan(&username, &password)
+
+	if err != nil {
+		pdb.logger.Error("get-custom-metrics-creds-from-credentials-table", err, lager.Data{"query": query})
+		return "", "", err
+	}
+	return username, password, nil
+}
+
+func (pdb *PolicySQLDB) ValidateCustomMetricTypes(appGUID string, metricsConsumer *models.MetricsConsumer) (bool, error) {
```

Comment:
why put ValidateCustomMetricTypes here?  No SQL statement in this implementation.

Reply:
Removed to custom_metrics_handlers.go